sbnobj/Common/SBNEventWeight/EventWeightParameterSet.h

```diff
@@ -56,7 +60,7 @@ struct EventWeightParameter {
 class EventWeightParameterSet {
 public:
   /** The type of random throws to perform. */
-  typedef enum rwtype { kMultisim, kPMNSigma, kFixed, kDefault } ReweightType;
+  typedef enum rwtype { kMultisim, kPMNSigma, kMultisigma, kFixed, kDefault } ReweightType;
```

Comment:
Chris already mentioned an issue with backward compatibility due to the fact that the new flag is added in the middle of the series.